### PR TITLE
Add language options on mobile

### LINKED ISSUE #93: ✨ SSW.com.au should have French and China flags in the mega menu (smaller screens)
Cc: @adamcogan @amankumarrr 

Hi team
  
### Pain
We don't have links to France / China on the menu - Tablet/Mobile view

### Suggested Solution

@adamcogan wants it after the SSW TV link as flags:

Image

Email thread: SSW.com.au should have French and China flags in the mega menu

### Tasks
- [x] 1. Add flags with links to ssw.fr and ssw.com.cn respectively 

Thanks!

### PR DESCRIPTION
**Pull request by Calinator444**
### Description

This PR updates the mega menu so you can switch between different versions (i.e. French, Chinese) of the site while on a mobile device. I've based the design on a concept designed by @bettybondoc. The search bar for mobile devices has been moved into the side bar so that the language options are immediately visible from the top of the screen.

- Fixed #93 

TLDR: I converted the search box into a dumb component, moved it into the side navigation bar for mobile devices and moved the form state and logic into the main layout component.

### Screenshots

state preserved between mobile and desktop view

**Figure**: **Search window state preserved between desktop and mobile view**
<br>

image

**Figure**: **New resized window layout**
<br>

image

**Figure**: **New sidebar layout**
<br>

image

**Figure**: **New sidebar open layout**

<br>

search test navbar

**Figure**: **Performing a search using the responsive layout**

### CODE REVIEW

**Review by tkapa — APPROVED** [review]
LGTM - After clarification from @jaydenalchin I think this solution has turned out well 👍
